src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1396,6 +1404,42 @@ class DocBaseViewer extends BaseViewer {
             default:
         }
     }
+
+    /**
+     * Gets the cached thumbnails toggled state based on file id. Will retrieve from
+     * localStorage if not cached. Defaults to true if not found.
+     * @return {boolean} Whether thumbnails is toggled open or not
+     */
+    getCachedThumbnailsToggledState() {
```

Comment:
Should this live in the Thumbnails Sidebar class?

Reply:
I'll do a follow on PR to see how I can clean this up